api/maven-api-core/src/main/java/org/apache/maven/api/package-info.java

```diff
@@ -60,12 +61,13 @@
  * <p><dfn>Artifact resolution</dfn> is the process of {@linkplain org.apache.maven.api.services.VersionResolver
  * resolving the version} and then downloading the file.</p>
  *
- * <p><dfn>Dependency collection</dfn> builds a graph of {@link org.apache.maven.api.Node} objects representing
+ * <p><dfn>Dependency collection</dfn> builds a graph of {@link org.apache.maven.api.Node} objects containing
  * all the dependencies.</p>
  *
- * <p>The <dfn>Dependency graph flattening</dfn> process in Maven involves reducing a complex,
- * multi-level dependency graph to a simpler list where only the most relevant version of each artifact
- * (based on groupId and artifactId) is retained, resolving conflicts and eliminating duplicates to ensure
+ * <p>The <dfn>Dependency graph flattening</dfn> process in Maven reduces a complex,
+ * multi-level dependency graph to an ordered list that can be turned into a classpath.
```

Comment:
This would have to be slightly updated wrt to recent changes.
See https://github.com/apache/maven/blob/master/api/maven-api-core/src/main/java/org/apache/maven/api/services/DependencyResolverResult.java#L81-L97

Reply:
good point. done